test/suite/test_rollback_to_stable06.py

```diff
@@ -129,20 +129,13 @@ def test_rollback_to_stable(self):
         else:
             self.assertGreaterEqual(upd_aborted + hs_removed + keys_removed, nrows * 4)
 
-        if not self.in_memory and self.key_format == 'i':
```

Comment:
Can you add the `Reverse modification depends on the on-disk data that is unstable` scenario in another test that we discussed while developing WT-9846?